_sass/components/_background-image.scss

```diff
@@ -1,8 +1,7 @@
 @media #{$breakpoint-sm} {
   #intro-header {
-    background-image: url('../../img/header.svg');
-    background-position: left center;
+    background-image: url('../../img/header.png');
```

Comment:
Yeah good catch, we've been using svgs for everything so haven't had to worry about it, but just got a 2x from Ryan and will update.